Sources/App/Core/SearchFilter/Filters/KeywordSearchFilter.swift

```diff
@@ -33,8 +35,16 @@ struct KeywordSearchFilter: SearchFilterProtocol {
         self.predicate = .init(
             operator: (expression.operator == .is) ?
                 .caseInsensitiveLike : .notCaseInsensitiveLike,
-            bindableValue: .value("%\(expression.value)%"),
+            bindableValue: .value(expression.value),
             displayValue: expression.value
         )
     }
+
+    var sqlIdentifier: SQLExpression {
+        predicate.sqlBind
+    }
+
+    var sqlBind: SQLExpression {
+        any(Search.keywords)
+    }
```

Comment:
Right now I've fixed this by making `sqlIdentifier` and `sqlBind` return "flipped" expressions. I.e. since both are `SQLExpression`s, it doesn't really make a difference which one is the bind value and which one the column expression, apart from them being misnamed.

It's probably best if we merge this as is for the moment and I'll follow up with a PR that renames both those protocol getters, to reduce the noise in this PR.